ocrd_tesserocr/crop.py

```diff
@@ -125,12 +125,12 @@ def process(self):
                     bin_bbox = image.getbbox()
                     if not bin_bbox:
                         # this does happen!
-                        LOG.debug("Ignoring region '%s' because its binarization is empty", ID)
+                        LOG.info("Ignoring region '%s' because its binarization is empty", ID)
```

Comment:
So, you decided to take the *more-verbose* road. We have to see if this is feasible in the long run.

Reply:
Sorry, then I misunderstood you. So you want this to stay in DEBUG here, and downgrade similar statements in `segment-region` to DEBUG as well?